frontend/src/components/MapView/LeftPanel/AnalysisPanel/AnalysisTable/index.tsx

```diff
@@ -73,22 +81,30 @@ const AnalysisTable = memo(
     const renderedTableHeaderCells = useMemo(
       () =>
         columns.map(column => (
-          <TableCell key={column.id} className={classes.tableHead}>
+          <TableCell
```

Comment:
Probably not a huge issue that’s worth sinking too much time into, but the current styles allow column widths to change between pages, which can feel a bit jarring.